docs/examples/values/prod/confluence.yaml

```diff
@@ -0,0 +1,375 @@
+  # directory in the volume contains the library file.
+  additionalLibraries: []
+#    - volumeName:
+#      subDirectory:
+#      fileName:
+
+  # -- Specifies a list of additional Confluence plugins that should be added to the Confluence container.
+  # These are specified in the same manner as the additionalLibraries field, but the files will be loaded
+  # as bundled plugins rather than as libraries.
+  additionalBundledPlugins: []
+#    - volumeName:
+#      subDirectory:
+#      fileName:
+
+  # -- Defines any additional volumes mounts for the Confluence container.
+  # These can refer to existing volumes, or new volumes can be defined in volumes.additional.
+  additionalVolumeMounts: []
+
+  # -- Defines any additional environment variables to be passed to the Confluence container.
+  # See https://hub.docker.com/r/atlassian/confluence-server for supported variables.
+  additionalEnvironmentVariables: []
+
+  jvmDebug:
+    # -- If set to true, Confluence JVM will be started with debugging port 5005 open.
+    enabled: false
+
+synchrony:
+  # -- True if Synchrony (i.e. collaborative editing) should be enabled.
+  # This will result in a separate StatefulSet and Service to be created for Synchrony.
+  # If disabled, then collaborative editing will be disabled in Confluence.
+  enabled: true
+  service:
+    # -- The port on which the Synchrony Kubernetes service will listen
+    port: 80
+    # -- The type of Kubernetes service to use for Synchrony
+    type: ClusterIP
+  ports:
+    # -- The port on which the Synchrony container listens for HTTP traffic
+    http: 8091
+    # -- The port on which the Synchrony container listens for Hazelcast traffic
+    hazelcast: 5701
+  readinessProbe:
+    # -- The initial delay (in seconds) for the Synchrony container readiness probe, after which the probe will start running
+    initialDelaySeconds: 5
+    # -- How often (in seconds) the Synchrony container readiness probe will run
+    periodSeconds: 1
+    # -- The number of consecutive failures of the Synchrony container readiness probe before the pod fails readiness checks
+    failureThreshold: 30
+  resources:
+    jvm:
+      # -- The minimum amount of heap memory that will be used by the Synchrony JVM
+      minHeap: "1g"
+      # -- The maximum amount of heap memory that will be used by the Synchrony JVM
+      maxHeap: "2g"
+      # -- The memory allocated for the Synchrony stack
+      stackSize: "2048k"
+    container: 
+      requests:
+        cpu: "2"
+        memory: "2.5G"
+  # -- The base URL of the Synchrony service.
+  # This will be the URL that users' browsers will be given to communicate with Synchrony, as well as the URL that the
+  # Confluence service will use to communicate directly with Synchrony, so the URL must be resovable both from inside and
+  # outside the Kubernetes cluster.
+  ingressUrl: confluence-synchrony.k8s.com 
```

Comment:
Does this work without an additional DNS setup? I thought it is usually `confluence-ingress/synchrony` e.g.: https://confluence.k8s.com/synchrony